### PR TITLE
added info about other cloud providers and token for 

### PR DESCRIPTION
**Pull request by ebgitelman**

### CODE REVIEW

File: product_docs/docs/biganimal/release/using_cluster/01_postgres_access/index.mdx [hunk 1]
```diff
@@ -98,7 +98,7 @@ If you use a single database to host multiple schemas, create a database owner a
     ```
 ## IAM authentication for Postgres
 
-Any AWS user with an AWS account connected to a BigAnimal subscription who has the Postgres role of iam_aws can authenticate to the database using their AWS IAM credentials.
+Any user with a cloud account connected to a BigAnimal subscription who has the Postgres IAM role iam_aws, iam_azure, or iam_gcp can authenticate to the database using their IAM credentials.
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/using_cluster/01_postgres_access/index.mdx:101]:
```suggestion
Any user with a supported cloud account connected to a BigAnimal subscription who has the Postgres IAM role iam_aws, iam_azure, or iam_gcp can authenticate to the database using their IAM credentials.
```

File: product_docs/docs/biganimal/release/using_cluster/01_postgres_access/index.mdx [hunk 2]
```diff
@@ -109,32 +109,37 @@ Provision your cluster before configuring IAM for Postgres.
     1. Select **Create Cluster** or **Save**.
     !!!note
     To turn on IAM authentication using the CLI, see [Using IAM authentication on AWS](/biganimal/latest/reference/cli/using_features/#iam-authentication-cli-commands).
-1. In AWS, get the ARN of each IAM user requiring database access. In the AWS account connected to BigAnimal, use AWS Identity and Access Management (IAM) to perform user management. See the [AWS documentation](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_manage.html). 
+1. From your cloud provider, get the user name of each IAM user requiring database access. In the cloud account connected to BigAnimal, use Identity and Access Management (IAM) to perform user management. See the [AWS documentation](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_manage.html).
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/using_cluster/01_postgres_access/index.mdx:112]:
```suggestion
1. From your cloud provider, get the user name of each IAM user requiring database access. In the cloud account connected to BigAnimal, use Identity and Access Management (IAM) to perform user management. 
```

File: product_docs/docs/biganimal/release/using_cluster/01_postgres_access/index.mdx [hunk 3]
```diff
@@ -109,32 +109,37 @@ Provision your cluster before configuring IAM for Postgres.
...
+
+For either method, you must first authenticate to your cloud service provider IAM to get your password or token.
 
 !!! Note 
     You can continue to log in using your Postgres username and password. However, doing so doesn’t provide IAM authentication even if this feature is configured.
 
-1. Using your AWS CLI or Cloud shell, obtain your ARN and access key. For guidance on obtaining your ARN and access key, see [Managing access keys for IAM users](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html).
+1. Get your credentials for your IAM-managed cloud account.
+    - For AWS, your password is your access key (access key id + secret access key). To get your access key, see [get-access-key-info](https://docs.aws.amazon.com/cli/latest/reference/sts/get-access-key-info.html) To get your authorization token, see [get-authorization-token](https://docs.aws.amazon.com/cli/latest/reference/ecr-public/get-authorization-token.html).
```
Comment (amrita-suresh) [inline product_docs/docs/biganimal/release/using_cluster/01_postgres_access/index.mdx:138]: @edbmanniwood is there a colon in between the access key id and secret access key?

Comment (edbmanniwood) [reply product_docs/docs/biganimal/release/using_cluster/01_postgres_access/index.mdx:138]: Hello, @amrita-suresh and @ebgitelman ! Yes, AWS users will want to place a colon between their aws access key ID and their aws secret access key (<aws_access_key_id>:<aws_secret_access_key>) and then they can use the resulting string of characters as their password.

Comment (ebgitelman) [reply product_docs/docs/biganimal/release/using_cluster/01_postgres_access/index.mdx:138]:
```suggestion
    - For AWS, your password is your access key (in the form <access key id>:<secret access key>). To get your access key, see [get-access-key-info](https://docs.aws.amazon.com/cli/latest/reference/sts/get-access-key-info.html) To get your authorization token, see [get-authorization-token](https://docs.aws.amazon.com/cli/latest/reference/ecr-public/get-authorization-token.html).
```

**Review by amrita-suresh — DISMISSED** [review]

File: product_docs/docs/biganimal/release/using_cluster/01_postgres_access/index.mdx [hunk 4]
```diff
@@ -109,32 +109,37 @@ Provision your cluster before configuring IAM for Postgres.
...
+
+For either method, you must first authenticate to your cloud service provider IAM to get your password or token.
 
 !!! Note 
     You can continue to log in using your Postgres username and password. However, doing so doesn’t provide IAM authentication even if this feature is configured.
 
-1. Using your AWS CLI or Cloud shell, obtain your ARN and access key. For guidance on obtaining your ARN and access key, see [Managing access keys for IAM users](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html).
+1. Get your credentials for your IAM-managed cloud account.
+    - For AWS, your password is your access key (in the form <access key id>:<secret access key>). To get your access key, see [get-access-key-info](https://docs.aws.amazon.com/cli/latest/reference/sts/get-access-key-info.html) To get your authorization token, see [get-authorization-token](https://docs.aws.amazon.com/cli/latest/reference/ecr-public/get-authorization-token.html).
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/using_cluster/01_postgres_access/index.mdx:138]:
```suggestion
    - For AWS, your password is your access key (in the form &lt;access key id>:&lt;secret access key>). To get your access key, see [get-access-key-info](https://docs.aws.amazon.com/cli/latest/reference/sts/get-access-key-info.html) To get your authorization token, see [get-authorization-token](https://docs.aws.amazon.com/cli/latest/reference/ecr-public/get-authorization-token.html).
```

File: product_docs/docs/biganimal/release/using_cluster/01_postgres_access/index.mdx [hunk 5]
```diff
@@ -109,32 +109,37 @@ Provision your cluster before configuring IAM for Postgres.
...
     ```
 
-1. For each IAM user, run this Postgres command:
+1. For each IAM user, run the `CREATE USER` Postgres command. For example, for AWS, use:
 
     ```
     CREATE USER "<ARN>" IN ROLE iam_aws;
     ```
 
+    Where <ARN> is the Amazon resource name. (For Azure, use the user principal name. For GCP, use the email address.)
```
Comment (ebgitelman) [inline product_docs/docs/biganimal/release/using_cluster/01_postgres_access/index.mdx:126]:
```suggestion
    Where &lt;ARN> is the Amazon resource name. (For Azure, use the user principal name. For GCP, use the email address.)
```

**Review by nidhibhammar — APPROVED** [review]

**Pull request merged**